script.js

```diff
@@ -1,12 +1,429 @@
 document.addEventListener('DOMContentLoaded', () => {
   document.querySelector('.logo__title').classList.add('rotate');
+  const modal = document.createElement('div');
+  const body = document.querySelector('body');
+  const contPrjctDet = document.createElement('section');
+  const contDetailwrapper = document.createElement('div');
+  const heading = document.createElement('h1');
+  const contList = document.createElement('ul');
+  const contListItem1 = document.createElement('li');
+  const contListItem2 = document.createElement('li');
+  const contListItem2I = document.createElement('i');
+  const contListItem3 = document.createElement('li');
+  const contListItem4 = document.createElement('li');
+  const contListItem4I = document.createElement('i');
+  const contListItem5 = document.createElement('li');
+  const figure = document.createElement('figure');
+  const imageCont = document.createElement('img');
+  const contListLang = document.createElement('ul');
+  const contListLang1 = document.createElement('li');
+  const contListLang2 = document.createElement('li');
+  const contListLang3 = document.createElement('li');
+  const langText = document.createElement('button');
+  const langText1 = document.createElement('button');
+  const langText2 = document.createElement('button');
+  const contButton = document.createElement('div');
+  const buttonLive = document.createElement('a');
+  const buttonSource = document.createElement('a');
+  const iconButtonLive = document.createElement('i');
+  const iconButtonSource = document.createElement('i');
+  const span1 = document.createElement('span');
+  const span2 = document.createElement('span');
```

Comment:
- [ ] OPTIONAL - You should aim to break up your code into functions. Running large blocks of code is really not modular and would make it hard to maintain.